cluster/src/main/java/com/hartwig/pipeline/report/FailureSummary.java

```diff
@@ -0,0 +1,38 @@
+package com.hartwig.pipeline.report;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.hartwig.pipeline.PipelineState;
+import com.hartwig.pipeline.StageOutput;
+import com.hartwig.pipeline.execution.PipelineStatus;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FailureSummary {
```

Comment:
I can see the reasoning (it's a summary showing any failures) but it's not necessarily a failure. Could we call it `StageSummary` or `ExecutionSummary` or something similar?

Reply:
See your point, going with VmExecutionLogSummary